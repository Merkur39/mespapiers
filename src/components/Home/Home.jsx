import React from 'react'

import { useI18n } from 'cozy-ui/react/I18n'

const Home = () => {
  const { t } = useI18n()

  return <p>{t('Home.welcome')}</p>
}

export default Home
