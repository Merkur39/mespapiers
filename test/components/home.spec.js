'use strict'

/* eslint-env jest */

import React from 'react'
import { render } from '@testing-library/react'

import AppLike from '../components/AppLike'
import Home from 'components/Home/Home'

describe('Home components:', () => {
  const setup = () => {
    return render(
      <AppLike>
        <Home />
      </AppLike>
    )
  }
  it('Home should be rendered correctly', () => {
    const { container } = setup()
    expect(container).toMatchSnapshot()
  })
})
