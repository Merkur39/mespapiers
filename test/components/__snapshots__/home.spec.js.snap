// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home components: Home should be rendered correctly 1`] = `
<div>
  <p>
    Wait&See ... Mes papiers
  </p>
</div>
`;
